src/v3/src/playground.ts

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint-disable no-alert,no-console */
+
+import { assertNoEnglishLeaks } from '../../../playground/LocaleUtils';
+import {
+  OktaSignInAPI,
+  OktaSignInConstructor,
+  RenderResult,
+  RenderResultSuccessNonOIDCSession,
+  WidgetOptions,
+} from '../../types';
+import { config } from '../widgetrc';
+import OktaSignIn from './OktaSignIn'; // FIXME should be set by cdn bundle
```

Comment:
Important FIXME here. `window.OktaSignIn` is implicitly tested here in the original playground script